content/pages/privacy/en.md

```diff
@@ -1,152 +1,153 @@
+The legal basis for this processing is our legitimate interest, according to Art. 6(1)(f) GDPR.
+
+**Legitimate interests:**
+
+Our legitimate interest is to provide the publish functionality to you.
 
 **Retention period:**
 
-deltaDAO discloses your wallet address and author name by transmitting it to a smart contract stored in machine-readable format permanently on-chain to enable an audit trail.
+Gaia-X AISBL discloses your wallet address and public key by transmitting it to a smart contract stored in machine-readable format permanently on-chain on the Gaia-X testnet to enable an audit trail.
 
-## 4.4 Showing data assets
+## 4.4 Caching and showing data services
 
-When you published a data asset, we retrieve, organize, and store your wallet address and author name on a metadata cache operated by deltaDAO. Moreover, we show your asset on our market.
+When you published a data service, we retrieve, organize, and store your wallet address on a metadata cache operated by Gaia-X AISBL. Moreover, we show your data service on our portal demonstrator.
 
 **Purposes:**
 
-Retrieving data from the blockchain is time-consuming. Hence, we retrieve, organize, and store your personal data on a cache to improve the performance of our market. We show your data asset on our market, so visitors can find, consume and stake in it.
+Retrieving data from the blockchain is time-consuming. Hence, we retrieve your personal data from the Gaia-X testnet and then organize and store it on a cache to improve the performance of our portal demonstrator. We show your data service including the wallet address and description you provided on our portal demonstrator, so visitors can find and consume it.
 
 **Legal basis:**
 
 The legal basis for this processing is our legitimate interest, according to Art. 6(1)(f) GDPR.
 
 **Legitimate interests:**
 
-Our legitimate interest is to enhance the user experience by improving the performance of our market.
+Our legitimate interest is to enhance the user experience by improving the performance of our portal demonstrator and to enable visitors to find and consume your data service.
 
 **Retention period:**
 
 Your wallet address and name are stored permanently on our metadata cache.
 
 ## 4.5 History table
 
-The history table of our market is a transparency tool that you can use to overview your transactions relating to our market. When you use the history table, we collect your wallet address. Then we retrieve the respective and relevant transactions stored on the blockchain (more precisely, from the metadata cache) and organize them in a table.
+The history table of our portal demonstrator is a transparency tool that you can use to overview your transactions relating to our portal demonstrator. When you use the history table, we collect your wallet address. Then we retrieve the respective and relevant transactions stored on the Gaia-X testnet (more precisely, from the metadata cache) and organize them in a table.
```

Comment:
question/issue(blocking): This table was previously located on a "history" page. Now this was changed to the `/profile` page, including the table. Do we need to change the wording accordingly?

E.g.: https://feat-mvp--mvg-portal-deltadao.netlify.app/profile
![grafik](https://user-images.githubusercontent.com/28757404/157031718-e29b8b1e-e517-480f-9d02-bc8355844dda.png)

Reply:
A change is not mandatory, but it is reasonable. I changed "history table" to "transaction overview".